src/providers/journal/journal.ts

```diff
@@ -26,21 +26,21 @@ export class JournalProvider {
     public networkStateProvider: NetworkStateProvider,
     private appConfigProvider: AppConfigProvider,
     private dateTimeProvider: DateTimeProvider,
-  ) {}
+  ) { }
 
   getJournal(lastRefreshed: Date): Observable<ExaminerWorkSchedule> {
     const staffNumber = this.authProvider.getEmployeeId();
     const journalUrl = this.urlProvider.getPersonalJournalUrl(staffNumber);
     const networkStatus = this.networkStateProvider.getNetworkState();
     if (lastRefreshed === null) {
       if (!this.authProvider.isInUnAuthenticatedMode() &&
```

Comment:
At risk of being pernickety here, isInUnAuthenticatedMode, drop the upper case 'A'.